packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -734,7 +734,7 @@ curl -v -X POST \
 
 To use Basic Auth, you would set `type` to `HEADER`, `key` to `Authorization`, and `value` to the Base64-encoded string of "username:password".
```

Comment:
To use Basic Auth,  set `type` to `HEADER`, `key` to `Authorization`, and `value` to the Base64-encoded string of "username:password".